### ISSUE TITLE
org-roam migration to v2 not working

### ISSUE THREAD

**Issue by PortStone** [hlissner/doom-emacs#5797]
### What did you expect to happen?

It migrated correctly my files from v1 to v2.

### What actually happened?

After I run `org-roam-migrate-wizard` I get `No node at point` and if I close Emacs it says `Save file /home/portstone/org/roam/20210509215114-msrp.org`

### Describe your attempts to resolve the issue

I deleted that file (`/home/portstone/org/roam/20210509215114-msrp.org`) since I didn't know what could be wrong with it. It runs and completes but still nodes are missing after I type `spc n r f.` I restart Emacs and type again `spc n r f` and it freezes at `Processing modified files....50%`

Content of `/home/portstone/org/roam/20210509215114-msrp.org`
```
#+title: MSRP
#+roam_alias: "RRP" "SRP"

MSRP stands for manufacturer's suggested retail price.
RRP stands for recommended retail price.
SRP stands for suggested retail price.
```

At first I did it like it said in the [documentation](https://github.com/hlissner/doom-emacs/blob/develop/modules/lang/org/README.org#migrating-your-existing-files-from-v1-roam-to-v2-roam2) but I thought reinstalling Doom Emacs completely will may fix it since may version of Doom Emacs was around half a year old. Therefore I deleted my `.emacs` folder and ran those two commands:
```
git clone --depth 1 https://github.com/hlissner/doom-emacs ~/.emacs.d
~/.emacs.d/bin/doom install
```

### Steps to reproduce

1. Delete .emacs folder
2. Make sure in init.el it is set to org roam2
4. Install Emacs (with those two commands)
5. Start Emacs and run `org-roam-migrate-wizard`
4. After it finished restart Emacs and use keybinding: `spc n r f`
5. Freezing at `Processing modified files...50%`

### System Information

https://pastebin.com/AAKQ6jKz

**Comment by PortStone** [comment]
> Does unpinning org-roam fix the issue? I.e.
> 
>     1. Add `(unpin! org-roam)` to `~/.doom.d/packages.el`
> 
>     2. Run `$ doom sync -u`
> 
>     3. Restart Emacs

No it still hangs. I also checked if it maybe because of my folder structure since I have tons of pictures and other files that I reference to. Therefore I deleted everything except the `.org` files but it keeps hanging at `Processing modified files...48%` (I don't know why it is now hanging at 48% and not 50%). 
It needs to convert `197` org files. I also don't have a feedback if it is finished with migrating. It just says `roam/20210509215114-example.org 33L, 200C written` and after some minutes it doesn't changed therefore I assume it finished since it is not saying something like `finished migrating`.
Also my process of migrating is that it outputs `roam/20210509215114-example.org 33L, 200C written` and after some seconds it does  `Processing modified files...done` and again does output `roam/20210509215114-another-example.org 33L, 200C written` for the rest of files it converted. Shouldn't it write first everything and after that process the files?
